cache.c

```diff
@@ -209,7 +209,7 @@ void cache_remove_ti(struct track_info *ti)
 static int read_cache(void)
 {
 	unsigned int size, offset = 0;
-	struct stat st = {};
+	struct stat st;
```

Comment:
This is the current OS X clang version (`Apple LLVM version 7.3.0 (clang-703.0.29)`). Let's just stick with `{}`, modern GCC should be OK with this (https://github.com/cmus/cmus/pull/428#issuecomment-204972047).